sprint1/signup.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable.css"
+    />
+    <link rel="stylesheet" href="assets/css/styles.css" />
+  </head>
+  <body class="login-body">
+    <!-- 로고 -->
+    <div class="big-logo">
+      <img src="/assets/images/logo.png" alt="판다마켓 로고" />
+      판다마켓
+    </div>
+    <!-- 회원가입 영역 -->
+    <div class="login-box">
+      <!-- 회원가입 -->
+      <div class="login-wrapper">
+        <div class="input-wrapper">
+          <div>
+            <p><label for="email">이메일</label></p>
```

Comment:
❗️ 수정요청
p 태그로 감싸지 않아도 html 구조나 디자인 구현에서 무리가 없을 것 같아요~

```suggestion
            <label for="email">이메일</label>
```